vital/types/class_map.h

```diff
@@ -110,7 +110,7 @@ class VITAL_EXPORT detected_object_type
    * @param class_name Class name
    * @param score Probability score for the class
    */
-  detected_object_type( const std::string& class_name,
+  class_map( const std::string& class_name,
```

Comment:
score seems to indented incorrectly now.

Reply:
Fixed in e45da5d1ba12fae92d21e0232281147b52860050. This happened in a different method declaration as well.